packages/cli/generation/ir-generator/src/examples/validateObjectExample.ts

```diff
@@ -19,7 +19,8 @@ export function validateObjectExample({
     exampleResolver,
     workspace,
     example,
-    breadcrumbs
+    breadcrumbs,
+    depth = 0
```

Comment:
thoughts on not defaulting `depth` so that each caller is forced to pass it in and make sure it can't randomly get reset to 0?

Reply:
Good catch, I agree with enforcing this.